services/discovery/discovery.go

```diff
@@ -48,6 +48,28 @@ func FromDependencies(deps resource.Dependencies, name string) (Service, error)
 }
 
 // Service describes the functions that are available to the service.
+//
+// For more information, see the [Discovery service docs].
+//
+// DiscoverResources example:
+//
+//	// Get the discovered resources of a Discovery Service.
+// 	cfgs, err := myDiscoveryService.DiscoverResources(ctx, nil)
+// 	if err != nil {
+// 		logger.Fatal(err)
+// 	}
+//  // Print out the discovered resources.
+// 	for _, cfg := range cfgs {
+// 		fmt.Printf("Name: %v\tModel: %v\tAPI: %v", cfg.Name, cfg.Model, cfg.API)
+// 		fmt.Printf("Attributes: ", cfg.Attributes)
+// 	}
+//
+// For more information, see the [discover resources method docs].
+//
+//
+// [Discovery service docs]: https://docs.viam.com/dev/reference/apis/services/discovery/
```

Comment:
this needs to point to the discovery service page

Reply:
It’s not terrible if it continues to point to the API page since the top of the API page will link over to the general service page once it exists. You’re right that would be more consistent though.